pages/api/globalSearch.ts

```diff
@@ -113,14 +129,37 @@ export default async function handler(req: Request) {
       ]
     }
   } else {
-    let { data } = await collectionQuery
+    // Get current usd prices for each chain
+    const usdCoinPrices = await fetch(`${HOST_URL}/api/usdCoinConversion`).then(
+      (res) => res.json()
+    )
 
-    searchResults = data.collections.map((collection: SearchCollection) => ({
-      type: 'collection',
-      data: collection,
-    }))
+    const responses = await Promise.all(promises)
+    await responses.forEach((response, index) => {
```

Comment:
Should we also call this async with all the other requests to speed up the response?